src/styles/main.scss

```diff
@@ -0,0 +1,17 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+
+  h2,
+  p {
+    margin: 0;
+  }
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
It's better to have a separate file for general styling and keep just imports in the main file